build.gradle.kts

```diff
@@ -126,6 +126,7 @@ subprojects {
       parallel = true
       buildUponDefaultConfig = false
       ignoreFailures = false
+      autoCorrect = true
```

Comment:
Should turn this off on CI. Auto-correct only makes sense locally where we get a chance to include the changes in the PR. I've marked the PR as draft for now and will come back to this.